react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import './ToggleSwitch.css';
+
+export default function ToggleSwitch() {
+  const [toggle, setToggle] = useState(false);
+
+  let text = toggle ? 'ON' : 'OFF';
+  let toggleClass = toggle ? 'is-on' : '';
+
+  return (
+    <div className={`toggle-switch ${toggleClass}`}>
+      <div className={`slider`} onClick={() => setToggle(!toggle)}>
```

Comment:
You should go back through your code and make sure it's "perfect".

In several of these, you are now just passing a hard-coded string as a prop. Those should use `prop="value"` instead of the string interpolation you once needed.